examples/src/test/resources/examples/osm_custom_params/oldApi-requestData.json

```diff
@@ -0,0 +1,67 @@
+{
+  "app" : "mapbox xyz old api",
+  "srs": "EPSG:3857",
+  "layout": "A4 landscape",
+  "outputFilename": "exemplemapbox",
+  "dpi": 72,
+  "outputFormat": "pdf",
+  "layers": [
+    {
+      "baseURL": "https://api.tiles.mapbox.com/v4/tmcw.map-7s15q36b/",
```

Comment:
I know it doesn't perfectly test mapbox but in the long run it is more reliable I think